packages/admin/admin-generator/src/index.ts

```diff
@@ -0,0 +1,2 @@
+export type { FormConfig, FormFieldConfig, GeneratorConfig, GridColumnConfig, GridConfig } from "./commands/generate/generate-command";
+export { CrudGeneratorConfig } from "./commands/generate/types";
```

Comment:
This belonged to the old generator, so it can be removed.

Reply:
was not sure about this. Removed it.